app/views/candidate_interface/gcse/year/edit.html.erb

```diff
@@ -1,16 +1,13 @@
-<% content_for :title, title_with_error_prefix(t("gcse_edit_details.page_titles.#{@subject}"), @application_qualification.errors.any?) %>
+<% content_for :title, title_with_error_prefix(t("gcse_edit_year.page_titles.#{@subject}"), @application_qualification.errors.any?) %>
 <% content_for :before_content, govuk_back_link_to(candidate_interface_gcse_details_edit_type_path) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_details_path, method: :patch do |f| %>
-      <h1 class="govuk-heading-xl"><%= t("gcse_edit_details.page_titles.#{@subject}") %></h1>
+    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_year_path, method: :patch do |f| %>
+      <h1 class="govuk-heading-xl"><%= t("gcse_edit_year.page_titles.#{@subject}") %></h1>
 
       <%= f.govuk_error_summary %>
```

Comment:
Here again, error summary should be above the `<h1>`.

Reply:
Resolved in 9046edb